### DISCUSSION TITLE
How to install WiX Toolset Build Tools v3 silently?

### DISCUSSION THREAD

**Discussion by SageGary** [wixtoolset/issues#7440] [Questions] [answered]
I need to install v3 of WiX Toolset Build Tools **silently** on a build server. Is there any documentation describing Command-Line Options for Installing the WiX Toolset v3? 
I am struggling to find any documentation for how to silently install `wix311.exe`.

**Comment by barnson** [comment, marked as answer]
Burn supports these switches: https://learn.microsoft.com/en-us/windows/win32/msi/standard-installer-command-line-options

**Reply by SageGary** [reply]
Thank you!
